docs/configuring-playbook-jitsi.md

```diff
@@ -134,19 +134,44 @@ By default a single JVB is deployed on the same hosts as the matrix server. To t
 There is a ansible role that can be run with the following tag:
 ` ansible-playbook -i inventory/hosts --limit jitsi_jvb_servers setup.yml --tags=common,setup-additional-jitsi-jvb `
 
-For this role to work you will need an additional section in the ansible hosts file with the details of the JVB hosts, for example 
+For this role to work you will need an additional section in the ansible hosts file with the details of the JVB hosts, for example:
 ```
 [jitsi_jvb_servers]
 <your jvb hosts> ansible_host=<ip address of the jvb host>
 ```
 
-The followings variables will need to be set in the vars file for each JVB:
+Each JVB will require a server id to be set so that it can be uniquely identify and this allows jitsi to keep track of which conferences are on which JVB.  
```

Comment:
```suggestion
Each JVB will require a server id to be set so that it can be uniquely identified and this allows Jitsi to keep track of which conferences are on which JVB.  
```